llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -597,6 +598,92 @@ class ConstantInt : public Constant {
 #endif
 };
 
+class ConstantFP final : public Constant {
```

Comment:
Why are we not adding ConstantData?  If we are shadowing LLVM IR why not do it rigorously?

Reply:
We can add it in, I will add a TODO.